debezium-quarkus/extensions/outbox/runtime/src/main/java/io/debezium/outbox/quarkus/internal/EventDispatcher.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.outbox.quarkus.internal;
```

Comment:
Note I moved this to an internal package, as users shouldn't interact with it directly.